tests/EdgeToEdge/Routes/AddDonationRouteTest.php

```diff
@@ -357,8 +358,9 @@ public function testGivenValidCreditCardData_showsIframeEmbeddingPage(): void {
 		);
 
 		$response = $client->getResponse();
-		$this->assertSame( 200, $response->getStatusCode() );
-		$this->assertContains( 'paytext_cc 3 12,34 € per Kreditkarte.', $response->getContent() );
+		$this->assertSame( Response::HTTP_FOUND, $response->getStatusCode() );
+		$this->assertContains( 'amount=1234', $response->getContent() );
+		$this->assertContains( 'Redirecting to', $response->getContent() );
```

Comment:
I think it would be worthwhile to assert what we actually want to happen - the location header on which the redirect will be based, not the fallback response body. Agree?

    $client->followRedirects( false );
    $client->request( ... );
    $response = $client->getResponse();
    $this->assertTrue( $response->isRedirect() );
    $this->assertContains( 'something.dot.somewhere', $response->headers->get( 'Location' ) );